src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3284,6 +3284,31 @@ internal sealed partial class LazyAnonymousFunctionExpression : BaseAnonymousFun
         public override IBlockOperation Body => SetParentOperation(_lazyBody.Value, this);
     }
 
+    internal sealed class FlowAnonymousFunctionExpression : Operation, IFlowAnonymousFunctionOperation
```

Comment:
>Expression [](start = 47, length = 10)

Name Operation, please. When I finally address the generation bug, standardizing all the classes will be one of the items I'll need to address, so if we can standardize all the new ones now it will be easier for me later.